INDEX

```diff
@@ -7,6 +7,14 @@
     "official": false,
     "primary_pkg": null
   },
+  "bind": {
+    "MANIFEST": "bind.json",
+    "description": "BIND DNS server",
+    "icon": "https://icons.freenas.org/community-icons/bind.png",
+    "name": "Bind DNS",
+    "official": false,
+    "primary_pkg": null
```

Comment:
As you're using the official `bind916` package from the FreeBSD package mirror, you could probably set the `primary_pkg` to `bind916` as well.

Reply:
Thank you for your review, the primary_pkg has been set.

Reply:
also the requested files have been added to the overlay dir and creation removed from post_install.sh

Reply:
Nice, good job with fast fixes :+1:  

The `/usr/local/etc/namedb/dynamic.local.db` this file doesn't seem to exist by the way so the [chown command](https://github.com/pandorixtech/iocage-plugin-bind/blob/master/post_install.sh#L12) in `post_install.sh` will not have any effect. Shouldn't this file exist or maybe remove the chown if it doesn't?

Reply:
sorry about that , when deleting extra content from post_install.sh I ended up making a typo mistake. Fixed now.
Thank you for your review :)